src/main/resources/network-modification.sql

```diff
@@ -47,6 +47,23 @@
         primary key (id)
```

Comment:
The file network-modification.sql is no more used.
It should have been discarded in PR : https://github.com/gridsuite/network-modification-server/pull/47
And the new changeset must be referenced in the db.changlog-master.yaml file